dts/arm/nuvoton/npcx/npcx7/npcx7-pinctrl.dtsi

```diff
@@ -111,42 +111,34 @@
 	/* PWM peripheral interfaces */
 	/omit-if-no-ref/ pwm0_gpc3: periph-pwm0 {
 		pinmux = <&alt4_pwm0_sl>;
-		drive-supported;
```

Comment:
The PWM nodes should set the `channel` property, correct?

Reply:
Yes, ```drive-supported``` is useless now. [Helper macro](https://github.com/zephyrproject-rtos/zephyr/pull/45712/files#diff-e1e5f8ebff804b124679e901b068da1bd8adf57c904dbbd6abd8c70597a89353R107) checks whether ```channel``` property exists. If so, the driver will create an item for pwm drive mode configuration automatically.